doc/articles/uno-development/ways-to-contribute.md

```diff
@@ -97,3 +97,12 @@ Once you're ready to create a PR, check out the [Guidelines for pull requests](.
 ### Need help?
 
 If there's anything you need, [ping us on our Discord Server](https://platform.uno/discord).
+
+## Sponsors & Grants
+Please consider sponsoring Uno Platform development, especially if your company benefits from this library.
+ 
+Your contribution will go towards adding new features and closing issues raised by community at Uno Platform backlog on GitHub (Issues · unoplatform/uno (github.com)) and making sure all functionality continues to meet our high quality standards.
+ 
+A grant for continuous full-time development has the biggest impact for progress. Periods of 2 to 5 days allow a contributor to tackle substantial complex issues which are otherwise left to linger until somebody can’t afford to not fix them.
```

Comment:
```suggestion
A grant for continuous full-time development has the biggest impact on progress. Periods of 2 to 5 days allow a contributor to tackle substantial complex issues that are otherwise left to linger until somebody can’t afford to not fix them.
```